curriculum/superblock-structure/full-stack.json

```diff
@@ -307,6 +307,7 @@
           "dashedName": "basic-regex",
           "blocks": [
             { "dashedName": "lab-regex-sandbox" },
+            { "dashedName": "lab-markdown-to-html-converter" },
```

Comment:
```suggestion
            { "dashedName": "lab-markdown-to-html-converter" },
            { "dashedName": "lab-regex-sandbox" },
```

Reply:
According to the order in the sheet:

![image](https://github.com/user-attachments/assets/be3c36ab-c3c8-4daf-9d8c-cbabada88c72)